tests/foreman/sys/test_fam.py

```diff
@@ -50,3 +62,20 @@ def test_positive_ansible_modules_installation(default_sat):
         assert doc_name == module_name
     # check installed modules against the expected list
     assert FOREMAN_ANSIBLE_MODULES.sort() == installed_modules.sort()
+
+
+@pytest.mark.tier1
+def test_positive_import_run_roles(sync_roles, default_sat):
+    """Import a FAM role and run the role on the Satellite
+
+    :id: d3379fd3-b847-43ce-a51f-c02170e7b267
+
+    :expectedresults: fam roles import and run successfully
+
+    """
+    roles = sync_roles.get('roles')
+    default_sat.cli.Host.ansible_roles_assign(
+        {'ansible-roles': roles, 'name': default_sat.hostname}
+    )
+    play = default_sat.cli.Host.ansible_roles_play({'name': default_sat.hostname})
+    assert "Ansible roles are being played" in play[0]['message']
```

Comment:
Please switch these to single-quote strings.

Reply:
I guess I need to be more consistent with my quotes...